src/Raven.Client/Documents/Changes/ChangeNotification.cs

```diff
@@ -246,6 +246,7 @@ public class TrafficWatchChange : DatabaseChange
         public object QueryTimings { get; set; }
     }
 
+    [Flags]
```

Comment:
This doesn't do what I expect you think it does.
It also not a good idea, because these are all mutually exclusive.